rollup.config.mjs

```diff
@@ -419,19 +432,29 @@ function templateCompilerConfig() {
   // need to be discovered from ember.debug.js instead when running in the
   // browser, and stubbed to ember-template-compiler.js in node.
   const externals = {
-    '@ember/template-compilation': `{ __registerTemplateCompiler(){} }`,
+    '@ember/template-compilation': `{
+      __esModule: true,
+      __registerTemplateCompiler(){},
+    }`,
     ember: `{
-      get ENV() { return require('@ember/-internals/environment').ENV },
-      get FEATURES() { return require('@ember/canary-features').FEATURES },
-      get VERSION() { return require('ember/version').default },
+      __esModule: true,
+      default: {
+        get ENV() { return require('@ember/-internals/environment').ENV },
+        get FEATURES() { return require('@ember/canary-features').FEATURES },
+        get VERSION() { return require('ember/version').default },
+      },
+    }`,
+    '@ember/-internals/glimmer': `{
```

Comment:
are these stubs that don't matter / aren't used in the actual output?

Reply:
They're used in the ember-template-compiler.js bundle. Which has always done wonky things so that it will share the main ember implementations of things if you load it into the browser, but bring its own copies of *some* of the things and stub the others if you load it by itself in node.

Reply:
The node tests accurately reflect the things that older versions of ember-cli-htmlbars actually expect the ember-template-compiler.js bundle to do.